projects/WTI-API/src/main/controllers/ContestController.java

```diff
@@ -566,6 +573,30 @@ public Response getStandings(
 					// we got the internal contest; pass it to the DefaultScoringAlgorithm and get back updated standings
 					try {
 						Properties props = internalContest.getContestInformation().getScoringProperties();
+
+						xlog(logger, "debug 22 props "+props);
```

Comment:
Is the intent to have all the xlog debug 22 messages in production?  If not, they should be removed.  To me, they clutter up the code.